client/app/shared/modals/model-edit/model-edit.component.ts

```diff
@@ -3,6 +3,8 @@ import { Editable, EditableModalService } from '../../models/interfaces/modal-mo
 import { Notifications } from '../../services/notifications.service'
 import { EditService } from '../../services/edit-service.service'
 
+declare var jQuery: any
```

Comment:
`jQuery` is not part of the package.json

Reply:
You're right. It's part of the Semantic pacakge. Thereafore, it only needs to be declared.